docs/qcfractal/source/managers_samples.rst

```diff
@@ -0,0 +1,191 @@
+Queue Manager Example YAML Files
+================================
+
+The primary way to set up a :term:`Manager` is to setup a YAML config file.
+This page provides helpful config files which mostly can be just copied
+and used in place (filling in things like :ref:`**username** and **password** <managers_server>`
+as needed.)
+
+The full documentation of every option and how it can be used can be found in
+:doc:`the Queue Manager's API <managers_config_api>`.
+
+For these examples, the ``username`` will always be "Foo" and the ``password`` will always be "b4R"
+(which are just placeholders and not valid).
+
+SLURM Cluster, Dask Adapter
+----------------------------
+
+To start a manager with a dask :term:`Adapter`, on a SLURM cluster, consuming 1 CPU and 8 GB of ram, targeting a Fractal
+Server running on that cluster, and using the SLURM partition ``default``, save the following YAML config file:
+
+.. code-block:: yaml
+
+    common:
+     adapter: dask
+     ntasks: 1
+     ncores: 1
+     memory: 8
+
+    server:
+     fractal_uri: "localhost:7777"
+     username: Foo
+     password: b4R
+
+    manager:
+     manager_name: "My First Manager"
+
+    cluster:
+     scheduler: slurm
+     walltime: "72:00:00"
+
+    dask:
+     queue: default
+
+
+Mutiple Tasks, 1 Cluster Job
+----------------------------
+
+This example starts a max of 1 cluster :term:`Job`, but multiple :term:`tasks<Task>`. The hardware will be
+consumed uniformly by each :term:`Worker`. With 8 cores, 20 GB of memory, and 4 tasks; each :term:`Worker` will get
+2 core and 5 GB of memory to work with. We set ``cluster.max_cluster_jobs`` to 1 to limit the number
+of jobs which can be started. Since this is SLURM, the ``squeue`` information will show this
+user has run 1 ``sbatch`` jobs which requested 4 cores and 20 GB of memory.
+
+.. code-block:: yaml
+
+    common:
+     adapter: dask
+     ntasks: 4
+     ncores: 8
+     memory: 20
+
+    server:
+     fractal_uri: "localhost:7777"
+     username: Foo
+     password: b4R
+
+    manager:
+     manager_name: "A multi-task manager"
+
+    cluster:
+     scheduler: slurm
+     walltime: "72:00:00"
+     max_cluster_jobs: 1
+
+    dask:
+     queue: default
+
+
+Testing the Manager Setup
+-------------------------
+
+This will test the :term:`Manager` to make sure its setup correctly, and does not need to
+connect to the :term:`Server`, and therefore does not need a ``server`` block.
```

Comment:
It might be good to link back to the Dask API documentation in these documents so that folks can see all options available.